### DISCUSSION TITLE
On trial adoption until desexed - reporting issues

### DISCUSSION THREAD

**Discussion by NQARSandra** [sheltermanager/asm3#1459] [Q&A / Help]
We have pups that are too young for desexing that are adopted, but with a desex bond which is refunded once they have been desexed. The movement remains 'trial adoption' despite the trial period finishing, until the pup is desexed and changed to 'adopted'.
This impacts on the 'time on shelter' - making it look as if we have had pups in care for much longer than they actually are.  Is there anyway to overcome this?

**Comment by Tetzor** [comment]
Hi, When the trial ends you need to uncheck the trial box in the movement (you must also set the end date if you haven't already), the time on shelter will then only include actual time at the shelter and exclude any time from the movement start date. If you do set a trial end date when you first create the movement, you will receive alerts when trial adoptions are ending to remind you to uncheck the trial box if the adoption is becoming final.

**Comment by NQARSandra** [comment]
Thanks Jon
Will this work and allow us to finalise an adoption even if the dog is not desexed?
Sent from my iPad

On 9 Apr 2024, at 7:12 pm, Jon ***@***.***> wrote:

﻿

Hi, When the trial ends you need to uncheck the trial box in the movement (you must also set the end date if you haven't already), the time on shelter will then only include actual time at the shelter and exclude any time from the movement start date. If you do set a trial end date when you first create the movement, you will receive alerts when trial adoptions are ending to remind you to uncheck the trial box if the adoption is becoming final.

**Comment by Tetzor** [comment]
You should always be able to finalise an adoption regardless of if the animal has been desexed. If you have the option "Warn when adopting an unaltered animal" turned on in Settings->Movements->Warnings, you will get a red warning. It won't prevent you making the adoption final, it is just to warn you.
